package.json

```diff
@@ -30,21 +30,23 @@
   },
   "homepage": "https://github.com/googlechromelabs/preload-webpack-plugin",
   "devDependencies": {
-    "babel-eslint": "^7.1.1",
-    "css-loader": "^0.28.7",
-    "eslint": "^3.14.1",
-    "eslint-config-google": "^0.7.1",
-    "extract-text-webpack-plugin": "^3.0.0",
-    "file-loader": "^0.11.2",
-    "html-webpack-plugin": "^2.26.0",
-    "jasmine": "^2.5.3",
-    "webpack": "^3.4.0"
+    "babel-eslint": "^8.2.2",
+    "css-loader": "^0.28.10",
+    "eslint": "^4.18.2",
+    "eslint-config-google": "^0.9.1",
+    "extract-text-webpack-plugin": "^4.0.0-beta.0",
+    "file-loader": "^1.1.11",
+    "html-webpack-plugin": "^3.0.4",
+    "jasmin": "0.0.2",
```

Comment:
that looks like it got added by mistake?

Reply:
Yes. That's a mistake.